# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../netpbm/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- stable/converter/ppm/xvminitoppm.c	2008/06/08 01:19:18	642
+++ stable/converter/ppm/xvminitoppm.c	2009/04/28 01:43:45	902
@@ -52,7 +52,7 @@
 
 
 static void
-getline(FILE * const ifP,
+getLine(FILE * const ifP,
         char * const buf,
         size_t const size) {
 
@@ -105,7 +105,7 @@
     int rc;
     bool endOfComments;
     
-    getline(ifP, buf, sizeof(buf));
+    getLine(ifP, buf, sizeof(buf));
 
     if (!STRNEQ(buf, "P7 332", 6))
         pm_error("Input is not a XV thumbnail picture.  It does not "
@@ -113,14 +113,14 @@
 
     endOfComments = FALSE;
     while (!endOfComments) {
-        getline(ifP, buf, sizeof(buf));
+        getLine(ifP, buf, sizeof(buf));
         if (STRNEQ(buf, "#END_OF_COMMENTS", 16))
             endOfComments = TRUE;
         else if (STRNEQ(buf, "#BUILTIN", 8))
             pm_error("This program does not know how to "
                      "convert builtin XV thumbnail pictures");
     }
-    getline(ifP, buf, sizeof(buf));
+    getLine(ifP, buf, sizeof(buf));
     rc = sscanf(buf, "%u %u %u", &cols, &rows, &maxval);
     if (rc != 3)
         pm_error("error parsing dimension info '%s'.  "

